src/args.rs

```diff
@@ -121,6 +121,10 @@ pub struct GenerateArgs {
     #[arg(short, long, action)]
     pub overwrite: bool,
 
+    /// Download git submodules (if there are any)
```

Comment:
```suggestion
    /// Skip download git submodules (if there are any)
```

this was presumably meant, right?